### PR TITLE
Revisions for hang in client session close.

### PR DESCRIPTION
**Pull request by dhaggart**
I've included fixes to tracing in this too; probably should have had them as a separate commit for clarity...

### CODE REVIEW

**Comment by dhaggart** [comment]
#build

**Comment by dhaggart** [comment]
#run-libby-bot

File: dev/com.ibm.ws.messaging.jms.2.0/src/com/ibm/ws/sib/api/jms/impl/JmsSessionImpl.java [hunk 1]
```diff
@@ -3008,7 +3031,7 @@ public void run() {
                         }
 
                         //store the owner JMSSession object in thread local for each and every Async Send operation.
-                        //in case if subsequent onComplete/OnException calls commit/rollback/clsoe on its own Seesion/connection/context
+                        //in case if subsequent onComplete/onException calls commit/rollback/clsoe on its own Seesion/connection/context
```
Comment (chackoj) [inline dev/com.ibm.ws.messaging.jms.2.0/src/com/ibm/ws/sib/api/jms/impl/JmsSessionImpl.java:3034]: s/clsoe/close/
s/Seesion/session/

File: dev/com.ibm.ws.messaging.jms.2.0/src/com/ibm/ws/sib/api/jms/impl/JmsSessionImpl.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2012, 2015 IBM Corporation and others.
+ * Copyright (c) 2012, 2015, 2020 IBM Corporation and others.
```
Comment (chackoj) [inline dev/com.ibm.ws.messaging.jms.2.0/src/com/ibm/ws/sib/api/jms/impl/JmsSessionImpl.java:2]: Maybe just "2012, 2020" for the copyright years

**Review by chackoj — APPROVED** [review]

**Pull request merged**